apps/speed.c

```diff
@@ -2557,7 +2557,9 @@ int speed_main(int argc, char **argv)
     }
 
     for (i = 0; i < loopargs_len; i++)
-        RAND_bytes(loopargs[i].buf, 36);
+        if(RAND_bytes(loopargs[i].buf, 36) <= 0){
```

Comment:
Style nit: missing white space after `if (`

Reply:
We don't use curly braces for single statement if blocks. And even if, there would be a space missing before the curly brace.